noxfile.py

```diff
@@ -32,6 +32,26 @@ def tests_impl(session, extras="socks,secure,brotli"):
     if session.python != "pypy" and session.python != "2.7":
         session.posargs.extend(["--random-order"])
 
+    # Only require unasync to run tests
+    import unasync  # please install unasync alongside nox to run tests
+
+    unasync.unasync_files(
+        [
+            "test/with_dummyserver/sync/__init__.py",
```

Comment:
Should this be `test/with_dummyserver/async/__init__.py`?

Reply:
Nice catch, thanks ! Fixed